src/frontend/src/pages/AdminToolsPage/ProjectsConfig/WorkPackageTemplateTable.tsx

```diff
@@ -0,0 +1,50 @@
+import { TableRow, TableCell, Box } from '@mui/material';
+import AdminToolTable from '../AdminToolTable';
+import { NERButton } from '../../../components/NERButton';
+import { isAdmin } from 'shared/src/permission-utils';
+import { useCurrentUser } from '../../../hooks/users.hooks';
+
+const WorkPackageTemplateTable = () => {
```

Comment:
Once the hook is completed we'll be able to call it similar to the linktype table. Though, for the hook we need an endpoint which is currently being done